src/config/config.example.ts

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
Ogólnie to tak, mamy dwie szkoły - albo robimy to poprzez `config.ts` albo poprzez `.env`. Osobiście preferuję zmienne środowiskowe. Możemy wtedy dodatkowo je zwalidować poprzez np. bibilotekę `joi`. Mogę przygotować taką propozycję jakby to wyglądało, jeśli @Keszua jesteś ciekawy.

Reply:
Zwróć uwagę, że z pliku config korzysta ormconfig, gdzie zdefiniowałem:
host: process.env.DB_HOST ?? config.dbHost, 
Czyli w pierwszej kolejności jest przypisywana wartość ze zmiennej  środowiskowej, gdy jej nie ma, to pobiera z config.
Kuba pokazywał taką metodę.

Jeśli masz lepsze rozwiązanie - jasne zajmij się tym.